### PR TITLE
fix(radio) - Fixes for radio and model YAML files.

### PR DESCRIPTION
**Pull request by philmoz**
Fixes issue from switch-refactor PR where old style trim names are not loaded from model files.

### CODE REVIEW

**Comment by 3djc** [comment]
Would that only work for T5 ? If you repurpose say T4, you could have TrimRudRight for example.

**Comment by gagarinlg** [comment]
> Would that only work for T5 ? If you repurpose say T4, you could have TrimRudRight for example.

You are saying that the name in the config file depends on the UI name of the trim switch?

**Comment by philmoz** [comment]
> Would that only work for T5 ? If you repurpose say T4, you could have TrimRudRight for example.

That's why I made it a draft - there is a lot more broken.
- Companion is broken (only recognises old names)
- Trim as source is broken in both radio and companion (name has changed)

**Comment by 3djc** [comment]
Not in new version, but pre ADC yes

> > Would that only work for T5 ? If you repurpose say T4, you could have TrimRudRight for example.
> 
> That's why I made it a draft - there is a lot more broken.
> 
> * Companion is broken (only recognises old names)
> * Trim as source is broken in both radio and companion (name has changed)



> > Would that only work for T5 ? If you repurpose say T4, you could have TrimRudRight for example.
> 
> You are saying that the name in the config file depends on the UI name of the trim switch?

Pre ADC:
customFn: 
   0:
      swtch: "TrimT5Down"
      func: OVERRIDE_CHANNEL
      def: "0,0,0"
   1:
      swtch: "TrimT6Up"
      func: OVERRIDE_CHANNEL
      def: "0,0,0"
   3:
      swtch: "TrimRudRight"
      func: OVERRIDE_CHANNEL
      def: "0,0,0"

**Comment by philmoz** [comment]
I have reverted all the trim names in the YAML files to the original values. Applies to trims as source or switch.

This means that no changes are required to Companion.

**Comment by philmoz** [comment]
I have added a fix for the wrong names being written to the radio.yml file for the pots and sliders.

**Comment by 3djc** [comment]
I'm not sure that's a right fix given other things that are in preparation

**Comment by philmoz** [comment]
> I'm not sure that's a right fix given other things that are in preparation

I'm trying to minimise the effort required to fix all the issues. This only changes the values saved to the YAML files, the UI names are not changed. Once it is stable the trim names can be updated in a future PR, including the Companion changes.

**Comment by 3djc** [comment]
Yeap, as a stopgap, it's perfect to me !

File: radio/src/storage/yaml/yaml_datastructs_funcs.cpp
Comment (3djc) [inline radio/src/storage/yaml/yaml_datastructs_funcs.cpp:1]: We cannot remove that part as there are already radio used with a firmware based on this, so the ability to read Tx(+-) needs to stay, even if it saves back to older format afterwards

**Comment by 3djc** [comment]
Thx @philmoz

**Pull request merged**